libraries/provider_install_package.rb

```diff
@@ -0,0 +1,65 @@
+class Chef
+  # Chef Provider for installing or removing Elasticsearch from package or source
```

Comment:
I'm not sure it makes sense to put examples here, since the class is never instantiated by a user (only by Chef, and the way it happens is quite convoluted). I'll improve the comment, though, as it could definitely be better.